ros2doctor/ros2doctor/api/__init__.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ros2doctor.api.platform import check_platform
+from ros2doctor.api.platform import print_platform_info
+from ros2doctor.api.platform import print_ros2_reqs
+
+
+def run_checks():
+    """Run all checks when `ros2 doctor/wtf` is called."""
+    check_platform()
```

Comment:
I think this function should use `iter_entry_points()` to get all checks rather than hard coding a list of them. That gives downstream users a way to add new checks specific to their packages without needing to change and make a new release of `ros2doctor`.

Reply:
Tried 
```suggestion
    for check in iter_entry_points('ros2doctor.api'):
        check.load()
```
load() doesn't seem to be invoking function call. Not sure how to call function within script after getting the iterator.

Reply:
I suspect `load()` returns the method, so `check.load()()` would call it with no arguments.